package.json

```diff
@@ -116,11 +116,14 @@
     "pa11y-ci": "^3.0.1",
     "prettier": "^2.3.1",
     "ts-node": "^10.9.1",
-    "typescript": "^4.8.4"
+    "typescript": "^4.9.5"
   },
   "overrides": {
     "@schematics/update": {
       "ini": "^1.3.8"
+    },
+    "ngx-mat-select-search": {
```

Comment:
There is a newer ngx-mat-select-search that works out of the box with Angular 16, but it uses the new Material components, and we're still on the legacy material components.